agent/index.js

```diff
@@ -1414,7 +1414,17 @@ ${regression}
       events.log.log,
       theme.dim(`  (this can take between 10 - 240 seconds)`),
     );
-    let newSandbox = await this.createNewSandbox();
+    // We don't have resiliency/retries baked in, so let's at least give it 1 attempt
+    // to see if that fixes the issue.
+    let newSandbox = await this.createNewSandbox().catch(() => {
+      this.emitter.emit(
+        events.log.log,
+        theme.dim(`  (double-checking sandbox availability)`),
+      );
+
+      return this.createNewSandbox();
```

Comment:
Somehow I don't see this log this part, but it's quickly throwing a `Max instances` error from the API 🤔 
> https://github.com/testdriverai/cli/actions/runs/16478965921/job/46588277179?pr=361